### ISSUE TITLE
Intermittent UI test failure - < DownloadFileTypesTest.downloadMultipleFileTypesTest>

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24415]
### Firebase Test Run:
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.f79aaa4b28c144f2/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.f604f88db8838613/test-cases)

### Stacktrace:
See Firebase links

**Comment by AndiAJ** [comment]
Failed 1x on 3/23 Main 
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.4693d8c7ff14109a/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.94a3dcb92ea02755/test-cases)

**Comment by sv-ohorvath** [comment]
A11y issue causing the failure was fixed, can be reopened if it fails again.
